tests/test_student.py

```diff
@@ -4,366 +4,79 @@
 from crank.views import *
 from crank.models import *
 
-v1 = "ab1234"       #valid uni
-v2 = "Alpha"        #valid first or last
-v3 = "Qwerty12345"  #valid password
-p = 302             #pass
-f = 200             #fail
+v1 = "fp12345"  # valid uni
+v2 = "Alpha"  # valid first or last
+v3 = "Qwerty12345"  # valid password
+v4 = "Qwerty12345"  # valid password
+
 
 class TestAccountCreate(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
-    def testSignUpValid(self):
```

Comment:
I don't think you want to be removing all the tests tony wrote? unless you 2 discussed it.